data/state/account.go

```diff
@@ -74,13 +74,13 @@ func (a *Account) GetNonce() uint64 {
 
 // SetBalanceWithJournal sets the account's balance, saving the old balance before changing
 func (a *Account) SetBalanceWithJournal(balance *big.Int) error {
-	entry, err := NewJournalEntryBalance(a, a.Balance)
+	entry, err := NewJournalEntryBalance(a, a.Balance.Get())
```

Comment:
indirectly, Balance in ProtoBigInt, ... anyway this will change with https://github.com/ElrondNetwork/elrond-go/pull/1041 and Balance will be big.Int .

Reply:
ok